### PR TITLE
Google cloud logo added

### LINKED ISSUE #577: Replace Google Cloud Logo with a new one
At the Diamond Sponsor Section, please replace the current google cloud photo with this new one. 

lockup_cloud_sm_h

### PR DESCRIPTION
**Pull request by abhishek71994**
resolves #577 . However, I do feel that the logo of Google cloud looks kind of small next to DAIMLER.

### CODE REVIEW

**Comment by abhishek71994** [comment]
Sorry about the other commits they haven't yet been resolved for another issue.

**Comment by mariobehling** [comment]
It seems you are making commits outside the issue description. It is great you are contributing. As you are making more contributions please also help reviewers and stick to best practices. Provide screenshot, test page and stick to relevant commits that match the description. Please read and follow the best practices https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/

**Comment by duttaditya18** [comment]
Please give a Preview Link. Here is a link anyway : https://rawgit.com/abhishek71994/2018.fossasia.org/Issues/index.html

File: css/theme-lava.css [hunk 1]
```diff
@@ -1066,8 +1066,13 @@ nav .container {
   opacity: 0.8 !important;
 }
 .video-header:before {
-  opacity: 0.5 !important;
-  background: #000 !important;
+  opacity: 0.7 !important;
+  background: -moz-linear-gradient(45deg, rgba(125,21,29,1) 0%, rgba(202,0,0,1) 50%, rgba(209,0,82,1) 85%, rgba(209,0,82,1) 100%); /* ff3.6+ */
+  background: -webkit-gradient(linear, left bottom, right top, color-stop(0%, rgba(125,21,29,1)), color-stop(50%, rgba(202,0,0,1)), color-stop(85%, rgba(209,0,82,1)), color-stop(100%, rgba(209,0,82,1))); /* safari4+,chrome */
```
Comment (duttaditya18) [inline css/theme-lava.css]: Why are these here?

File: index.html [hunk 2]
```diff
@@ -7815,6 +7815,10 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
                             <div class="col-sm-5">
                                 <input class="form-email signup-email-field" placeholder="Enter your email address" type="text" name="email" id="email">
                             </div>
+                            <div style="display:none;">
+                                <label for="hp">HP</label><br/>
+                                <input type="text" name="hp" id="hp">
```
Comment (duttaditya18) [inline index.html]: These?

**Comment by abhishek71994** [comment]
Made the commit changes.

**Comment by simsausaurabh** [comment]
@abhishek71994 use ```class="partner-logo"``` in ```img``` tag, this particular style is made for making logos comparable in size.

File: index.html [hunk 3]
```diff
@@ -6983,7 +6983,7 @@ <h1>Diamond Sponsors</h1>
                     <div class="col-md-4 col-sm-6 sponsor-column">
                         <div class="sponsor text-center">
                             <a href="https://www.google.com/cloud/" target="_self">
-                                <img alt="Google" src="img/GoogleCloud.png">
+                                <img alt="Google" src="img/google-cloud-logo.png">
```
Comment (simsausaurabh) [inline index.html]: @abhishek71994 use ```class="partner-logo"```, this particular style is made for making logos comparable in size.

Comment (abhishek71994) [reply index.html]: @simsausaurabh adding partner-logo as class is not working.

Comment (simsausaurabh) [reply index.html]: @abhishek71994 You will have to remove sponsor class from the div just above containing img tag. Then, it should work.

**Comment by mariobehling** [comment]
Resolved it already. Cheers.
